test/functionalTest/cases/950_GET_v2_entity/GET_v2_entity_complex.test

```diff
@@ -0,0 +1,356 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I'm wondering whether this test is really needed as entity rendering has already been tested by /v2/entities tests